remote/advancedcontainers/sharing-git-credentials.md

```diff
@@ -66,7 +66,8 @@ if [ -z "$SSH_AUTH_SOCK" ]; then
         # Launch a new instance of the agent
         ssh-agent -s &> $HOME/.ssh/ssh-agent
    fi
-   eval `cat $HOME/.ssh/ssh-agent`
+   eval `cat $HOME/.ssh/ssh-agent` > /dev/null
+   ssh-add 2> /dev/null
```

Comment:
@bedaberner per suggestion from @roblourens:

```suggestion
   ssh-add $HOME/.ssh/github_rsa 2> /dev/null
```

Reply:
I think doing it like this is worse, as it introduces value that you need to change based on your individual configuration hidden inside a snippet 99% of people are just going to copy. Since we mute the output of ssh-add it will not be easily noticable to users that ssh-add could not add the key that does not exist on their system.

Especially because of the wording above, people will just copy the whole snippet.

> Then add these lines to your ~/.bash_profile or ~/.zprofile (for Zsh) so it starts on login:

In this context, just adding all keys is preferable, because it will work without user changes.
maybe we can add a commented version on how to just add a single key?

```suggestion
   ssh-add 2> /dev/null
   # ssh-add ~/.ssh/<your ssh key> 2> /dev/null # use this syntax to add a specific key
```

Regardless if you insist on adding a specific key, we should at least make it obvious that there is user input required by marking the key with <> (which should be done in the example above anyway, seeing as `github_rsa` is not created in the scope of this example) and maybe adding some text.

Reply:
The default behavior should be "least privilege", so not in favor of only adding a comment to add a specific key.
Agree that the text should be updated to indicate that users have to add their specific key. 
Do you want to update the text, or otherwise you can remove the ssh-add updates from this PR and I'll create a follow-up PR.

Reply:
@ntrogh I've added a commit with some text.
I also changed `github_rsa` to `<your ssh key>` in both snippets to keep things consistent and make clear this requires user input.